src/coreclr-debug/activate.ts

```diff
@@ -48,7 +48,7 @@ async function checkForInvalidArchitecture(platformInformation: PlatformInformat
     return false;
 }
 
-async function completeDebuggerInstall(platformInformation: PlatformInformation, eventStream: EventStream): Promise<boolean> {
+export async function completeDebuggerInstall(platformInformation: PlatformInformation, eventStream: EventStream): Promise<boolean> {
```

Comment:
>export [](start = 0, length = 6)

is this necessary? am I missing something?

Reply:
oops, leftover from when I created the class in another file.

Reply:
And I need it again because I moved the class to a different .ts file.